### ISSUE TITLE
Consolidate Thai extractive question answering datasets

### ISSUE THREAD

**Issue by cstorm125** [vistec-AI/thai2transformers#55]
Consolidate all Thai QA datasets into one benchmark. This is done to have sizeable sample to train Thai QA.
Training sets (checklist of availability on huggingface datasets):

- [x] thaiqa_squad
- [x] iapp_wiki_qa_squad
- [x] xquad 

Test set:
- [x] iapp_wiki_qa_squad test set; removing questions with contexts overlapping with training sets

Available on huggingface datasets but cannot be used for Thai extractive QA:
- tydiqa
- mkqa

**Comment by cstorm125** [comment]
done
